src/inputgroup/skin.css

```diff
@@ -86,3 +86,89 @@
     }
   }
 }
+
+/* 
+  TODO: discuss with other 
+  Way to keep this datapicker related
+  IN PROGRESS
+*/
+.spectrum-InputGroup {
+  &.spectrum-Datepicker {
+    /** Normal **/
+    &:not(.spectrum-InputGroup--quiet) {
+      border: 1px solid var(--spectrum-textfield-border-color);
+    }
+    /** Quiet **/
+    &.spectrum-InputGroup--quiet {
+      border-bottom: 1px solid var(--spectrum-textfield-border-color);
+      hr {
+        left: calc(50% - 21px); /*22px half button width and 5px hald of HR width*/
+      }
+    }
+    /* Border color in different cases: hover, focus, invalid */
+    &:not(.is-disabled):hover,
+    &.is-focused {
+      .spectrum-InputGroup-field::placeholder {
+        color: var(--spectrum-textfield-placeholder-text-color-hover);
+      }
+      hr {
+        background-color: var(--spectrum-textfield-placeholder-text-color-hover);
+      }
+    }
+    &:not(.is-disabled):not(.is-focused):not(.is-invalid):hover {
+      border-color: var(--spectrum-textfield-border-color-hover);
+    }
+    &.is-focused {
+      border-color: var(--spectrum-textfield-border-color-down);
+    }
+    &.is-invalid:not(.is-focused) {
+      border-color: var(--spectrum-textfield-border-color-error);
+    }
+    &.is-disabled {
+      hr {
+        background-color: var(--spectrum-textfield-text-color-disabled);
+      }
+    }
+    /* Single & Range Inputs */
+    .spectrum-InputGroup-field {
+      border: 0;
+      &:focus ~ .spectrum-FieldButton {
+        border-color: var(--spectrum-textfield-border-color);
+        box-shadow: none;
+      }
+    }
+    /* Single & Range Button */
+    .spectrum-FieldButton {
+      border-top: 0;
+      border-right: 0;
+      border-bottom: 0;
+    }
+    /* Horisontal line */
+    /* TODO: Add class to HR */
+    hr {
+      background-color: var(--spectrum-textfield-placeholder-text-color);
+      border: 0;
+      margin: 15px 0px;
+      height: 1px;
+      width: 10px;
+      position: absolute;
+      top: 0;
+      left: calc(50% - 27px); /*22px half button width and 5px hald of HR width*/
+    }
+    /* Range */
+    /* TODO: use defined variable for width and padding */
+    &.spectrum-Datepicker--range {
+      /* Inputs */ 
+      .spectrum-InputGroup-field {
+        width: 100px;
+        min-width: 100px;
```

Comment:
These widths don't seem appropriate. We would hope that the minimum width for each input would relate to the date format being used, which may not be able to be expressed here (fyi @GarthDB )

To that I would suggest leaving this out since textfields default min-width should already be set to 72px.

Ultimately we would hope that the spacing between the HR and the inputs always be the mid-way point, even if the field has a fixed width.
![image](https://user-images.githubusercontent.com/13972198/49836730-fe4dcf00-fd60-11e8-80b7-8e30323c0da7.png)

Reply:
@NateBaldwinDesign Adjusting the margin of the em dash based on the text width of the first input's value or placeholder requires a javascript method to execute when the value changes. Should we add a utility method to the docs in order to do this?